app/forms/contact_form.rb

```diff
@@ -5,7 +5,7 @@ class ContactForm < Form
   before_validation do
     self.phone_number = phone_number.gsub(/[^0-9a-zA-Z]/, '')
   end
-  validates :phone_number, format: { with: /\d{10}/, message: proc { I18n.t('validations.phone_number') },
+  validates :phone_number, format: { with: /\A\d{10}\z/, message: proc { I18n.t('validations.phone_number') },
```

Comment:
I'd say we should check to see if we care about non-10-digit numbers (internationals) before committing to this

Reply:
And possibly, actually, if we ARE limiting to 10digit, maybe we want to clarify that in the copy....that's a design question.